packages/x-components/src/design-system/components/tag/default.tokens.scss

```diff
@@ -6,6 +6,13 @@
   --x-color-background-tag-default-selected: var(--x-color-base-neutral-95);
   --x-color-border-tag-default-selected: var(--x-color-border-tag-default);
   --x-color-text-tag-default-selected: var(--x-color-text-tag-default);
+  // color curated
+  --x-color-background-curated-tag-default: var(--x-color-base-neutral-100);
```

Comment:
Is the naming fine? `x-color-background-curated-tag-default` means that we are treating `curated-tag` as a component (and therefore it should have their own styles in its own folder) while I think `curated` is a modifier.

However treating `curated` as a modifier and naming it `x-color-background-tag-default-curated` brings the problem of combining it with the other `selected` modifier.

Reply:
the name should be:

`--x-color-background-tag-default-curated-selected`
`--x-color-background-tag-default-curated`